synthtool/gcp/templates/node_library/.kokoro/populate-secrets.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+# Populates requested secrets set in SECRET_MANAGER_KEYS from service account:
+# kokoro-trampoline@cloud-devrel-kokoro-resources.iam.gserviceaccount.com
+SECRET_LOCATION="$KOKORO_GFILE_DIR/secret_manager"
+mkdir -p $SECRET_LOCATION
+export CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE=${KOKORO_GFILE_DIR}/kokoro-trampoline.service-account.json
```

Comment:
TIL about this envvar `CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE`

Reply:
I learned about this today too, it felt risky to run `gcloud auth`, as we're going to then be running tests in the context of a different service account ... it feels like a recipe for bugs.

Reply:
It turns out that anything with CLOUDSDK_AUTH just aliases the flag, so it seems better to use ` --credential-file-override`.